atkinson/logging/logger.py

```diff
@@ -0,0 +1,43 @@
+#! /usr/bin/env python
+"""Interface to get a configured logger"""
+
+import importlib
+
+
+def getLogger(config=None):
+    """
+    Returns a logger object of the type specified.
+
+    :param config: Dictionary of values to configure the desired logger object.
+
+    For example, to use the default python logger, you can
+    either simply pass None, or specify any details supported by
+    the atkinson default driver for the python logger.  You
+    would specify the default logger (with custom loglevel) with:
+    ::
+
+      {'driver':'atkinson.logging.drivers.default',
+      'root':{'level':DEBUG}}
+
+    The atkinson logging wrapper code specifies a toplevel element in the
+    config dict of 'driver', pointing at the driver that will load the
+    desired configuration and return a logger to use.
+
+    Each driver must implement getLogger, so we can invoke it the same way from
+    this interface. In theory, drivers could be released as separate modules.
+
+    """
+    if not config:
+        config = _default_config()
+    try:
+        module = importlib.import_module(config.get('driver'))
+    except ModuleNotFoundError as e:
+        logger = getLogger(_default_config())
+        logger.error('Loading the default driver, {0}'.format(e.msg))
+        return logger
+
+    return module.getLogger(config)
```

Comment:
Minor nit, this line could be moved up into the try block since module does not really have context out side of that block. If an exception is thrown, it would be caught on the import_module call skipping the return call all together.

Reply:
Sure thing, moved it up, and tests are passing